recipes/riboseq-rust/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "riboseq-rust" %}
+{% set version = "1.2" %}
+{% set sha256 = "2b50915fda23f9bc9c8170d82fa5255ef2c3954f0d6a5ea15649b119ff506d0c" %} 
+
+package:
+  name: riboseq-rust
+  version: "1.2"
+
+source:
+  fn: RUST_{{ version }}.tar.gz
+  url: https://lapti.ucc.ie/rust/scripts/RUST_{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True # [not py27 or osx]
+
+requirements:
+  build:
```

Comment:
Oh, and we actually don't need any `build` requirements 😄 as we only copy (+some `sed`/`chmod`ing).

Reply:
Thanks for pointing that out :-)
Removed the build dependencies